src/awkward/_v2/_connect/cling.py

```diff
@@ -1424,6 +1424,8 @@ def from_form(cls, form, flatlist_as_rvec):
 
     def __init__(self, contents, fields, identifier, parameters, flatlist_as_rvec):
         self.contents = tuple(contents)
+        # FIXME: attribute in Numba
+        self.contenttypes = self.contents
```

Comment:
`RecordLookup` `tolayout()` expects a `contenttypes` attribute. I'm not sure if we need both `contents` and `contenttypes` here.